pkg/api/query/v1.go

```diff
@@ -152,6 +158,8 @@ func (qapi *QueryAPI) Register(r *route.Router, tracer opentracing.Tracer, logge
 	r.Get("/rules", instr("rules", NewRulesHandler(qapi.ruleGroups, qapi.enableRulePartialResponse)))
 
 	r.Get("/metadata", instr("metadata", NewMetricMetadataHandler(qapi.metadatas, qapi.enableMetricMetadataPartialResponse)))
+
+	r.Get("/exemplars", instr("exemplars", NewExemplarsHandler(qapi.exemplars, qapi.enableExemplarPartialResponse)))
```

Comment:
Let's be consistent with the upstream PR https://github.com/prometheus/prometheus/pull/6635/files#diff-9bcb876da5d17ef1046ab5ea531bbaaf64d753a7beaa2cb11a3fe6dd94e16563R349.

The endpoint is `query_exemplars`